### PR TITLE
Ambr 662

### PR DESCRIPTION
**Pull request by fcabrales-plos**
AMBR-662 adding syndication integration tests

### CODE REVIEW

**Comment by mcbrandt13** [comment]
looks fine

File: test/api/RequestObject/articlecc.py [hunk 1]
```diff
@@ -61,6 +64,21 @@ def add_article_revision(self, expected_response_code):
...
+      POST /articles/{doi}/revisions/1/syndications?syndicate
+      :param doi
+      :param revision
+      :param ingestion
+      :param syndication target
+      """
+      da_data = json.dumps({
+        "targetQueue": "activemq:plos." + syndication_target
+      })
+      response = self.doPostData('%s/%s/syndications?syndicate' % (ARTICLE_REVISION_API,REVISION), da_data, CONTENT_HEADERS)
```
Comment (slimeate) [inline test/api/RequestObject/articlecc.py]: Does running this test actually result in articles being syndicated?

File: test/api/RequestObject/articlecc.py [hunk 2]
```diff
@@ -27,15 +27,18 @@
...
 from ..resources import *
 
 from test.Base.api import needs
 from test.Base.MySQL import MySQL
-from ..resources import NOT_SCAPE_RELATED_ARTICLE_DOI
+from ..resources import NOT_SCAPE_RELATED_ARTICLE_DOI, CONTENT_HEADERS
 
 ARTICLE_API = API_BASE_URL + '/articles/'
 ARTICLE_REVISION_API = ARTICLE_API + RELATED_ARTICLE_DOI + '/revisions'
+HEADER = '-H'
```
Comment (slimeate) [inline test/api/RequestObject/articlecc.py]: this doesn't appear to be used

**Comment by slimeate** [comment]
Oh, I see this has already been merged . . .

**Pull request merged**